using-mobile-extensions/mobile-core/identity/identity-event-reference.md

```diff
@@ -4,8 +4,160 @@
 
 The following events are handled by the Identity extension:
 
+### Analytics Identity Response
+
+#### Event Details
+
+| Type                          | Source                                 | Paired | Direction                                                    |
+| :---------------------------- | :------------------------------------- | :----- | :----------------------------------------------------------- |
+| com.adobe.eventType.analytics | com.adobe.eventSource.responseIdentity | True   | This is the end of the paired [Event: v5 Analytics Request Identity](https://wiki.corp.adobe.com/display/ADMSMobile/Event%3A+v5+Analytics+Request+Identity) event containing the AID value |
+
+#### Event Description
+
+This event is a response from the Analytics module that contains the unique tracking identifier. This value may be null if identity module is enabled and there is a valid MID generated.
```

Comment:
This event is a response from the Analytics extension ...
... may be null if the Identity extension is enabled and there is a valid ECID generated.